fdbserver/DataDistributionQueue.actor.cpp

```diff
@@ -1416,6 +1451,11 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueueData* self,
 						}
 
 						bestTeams.emplace_back(bestTeam.first.get(), bestTeam.second);
+
+						if (rd.operation_ == RelocateOperation::MULTIPLY_REPLICAS &&
+						    SERVER_KNOBS->DYNAMIC_REPLICATION_ENABLED) {
```

Comment:
You shouldn't have to check the knob condition here. This kind of operation could only have been enqueued if the the knob was enabled.

Reply:
remove extra checking on `DYNAMIC_REPLICATION_ENABLED`